### DISCUSSION TITLE
How to choose the default locale for Filament? editing "locale" in config/app.php dindn't work

### DISCUSSION THREAD

**Discussion by dev-dev-15** [filamentphp/filament#11027] [Help]
The default language of Filament now is in English, I want to change it to Arabic so I changed the value of `locale` in `config/app.php` but the language in Filament is still English!

The locale outside Filament is "ar" which is expected, but in Filament routes the locale is "en".

I have the translation files of Filament Arabic.

I use `bezhanSalleh/filament-language-switch` and I change the language using it without any problem.

![image](https://github.com/filamentphp/filament/assets/14108109/3fe81fbd-79d9-4546-b518-7068d96372ae)

![image](https://github.com/filamentphp/filament/assets/14108109/03bdfe0c-e1a6-442e-a13c-c94f918acd2d)

![image](https://github.com/filamentphp/filament/assets/14108109/e44e33c6-73c5-4773-9915-e996e23c63be)

![image](https://github.com/filamentphp/filament/assets/14108109/1c4f4615-e288-4421-bdcf-4e89876a7d42)

**Comment by leandrocfe** [comment]
run `php artisan config:clear`

**Reply by dev-dev-15** [reply]
thanks but the configuration is not cached, the value of `locale` is `ar` in normal routes, just on Filament routes it's `en`

**Reply by boulama** [reply]
for anyone else who might need an answer:
the value in `.env` file takes priority to the one in `app.php`.
so, just set `APP_LOCALE=ar` in `.env`,

**Comment by begueradj** [comment]
Running into the same issue after I did the above changes.
